src/util.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const randomBytes = require('random-bytes')
```

Comment:
it is what random-bytes uses underneath

What about we expose this feature in libp2p-crypto?